census_view.hpp

```diff
@@ -134,6 +134,14 @@ class CensusView final
     wxDataViewCtrl* list_window_;
     wxObjectDataPtr<CensusViewDataViewModel> list_model_;
 
+    struct header_cache
+    {
+        std::string name;
+        bool        show;
+    };
+
+    std::vector<header_cache> all_headers_;
```

Comment:
Let's also rename this one to `current_headers_` to distinguish it better from `all_header_names` local variable in the code above.